How to/Add water mark annotation in PDF viewer/Web Service/Controllers/PdfViewerController.cs

```diff
@@ -91,12 +91,54 @@ public IActionResult Load([FromBody] Dictionary<string, string> jsonObject)
             return Content(JsonConvert.SerializeObject(jsonResult));
         }
 
+        [AcceptVerbs("Get")]
```

Comment:
why you have changed here?

Reply:
added the method to get the ajax request after the load method , where to add this method is it need to add at the end ?